libs/media-player/src/lib/MediaPlayer.tsx

```diff
@@ -32,25 +38,36 @@ export function MediaPlayer({ audioUrl, listenMessage }: MediaPlayerProps) {
         audioElement.addEventListener('canplaythrough', () => {
             setCanPlayThrough(true);
         });
+        audioElement.addEventListener('ended', handleAudioNotPlaying);
         setAudio(audioElement);
+
+        return () => {
+            audioElement.removeEventListener('ended', handleAudioNotPlaying);
+        };
     }, [audioUrl]);
 
     return (
         <div className={styles['container']}>
             {audio && (
                 <>
-                    <PlayCircleFilledIcon
-                        className={styles['media-controls']}
-                        onClick={() => safePlay()}
-                    >
-                        Play
-                    </PlayCircleFilledIcon>
-                    <PauseCircleFilledIcon
-                        className={styles['media-controls']}
-                        onClick={() => audio.pause()}
-                    >
-                        Pause
-                    </PauseCircleFilledIcon>
+                    {!isPlaying && (
```

Comment:
This is not a good way to do this. We never want to attempt to render `false` as a React node (`null`, however, is acceptable). In this case, I'd recommend:
```tsx
{
isPlaying ? <PlayCircleFilledIcon /> : <PauseCircleFilledIcon />
}
```